schema/metrics_config.schema.json

```diff
@@ -1,39 +1,39 @@
 {
     "$id": "metrics_config.schema.json",
     "$schema": "https://json-schema.org/draft/2020-12/schema",
-    "description": "The `metrics` section describes where to send metrics on extractor performance for remote monitoring of the extractor. We recommend sending metrics to a [Prometheus pushgateway](https://prometheus.io/), but you can also send metrics as time series in the CDF project.",
+    "description": "The `metrics` section describes where to send metrics on extractor performance for remote monitoring of the extractor. We recommend sending metrics to a [Prometheus pushgateway](https://prometheus.io/). You can also send metrics as time series in the CDF project.",
     "type": "object",
     "properties": {
         "push-gateways": {
             "type": "array",
-            "description": "List of prometheus pushgateway configurations",
+            "description": "List of Prometheus pushgateway configurations",
             "items": {
                 "type": "object",
-                "description": "The push-gateways sections contain a list of metric destinations.",
+                "description": "The pushgateways sections contain a list of metric destinations.",
```

Comment:
This should be hyphenated, it's the name of a config parameter, and the extractors will not recognize it as one word
``` suggestion
                "description": "The `push-gateways` sections contain a list of metric destinations.",
```

Reply:
@mathialo My bad, I read it as pushgateways.